packages/paste-website/src/pages/foundations/content/word-list/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Words list
+description: List of visual attributes (background colors, spacing, etc.) that our design language is based on.
+slug: /foundations/content/word-list/
+---
+
+import {Box} from '@twilio-paste/box';
+import {SidebarCategoryRoutes} from '../../../../constants';
+import {WordsList} from '../../../../components/word-list';
+
+# Word List
```

Comment:
suggestion: it'd be better to use <GenericHeader> here, for consistency

Reply:
done